src/Microsoft.Data.SqlClient/tests/FunctionalTests/SqlBulkCopyColumnOrderHintCollectionTest.cs

```diff
@@ -0,0 +1,579 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using Xunit;
+
+namespace Microsoft.Data.SqlClient.Tests
+{
+    public class SqlBulkCopyColumnOrderHintCollectionTest
+    {
+        private static SqlBulkCopyColumnOrderHintCollection CreateCollection() => new SqlBulkCopy(new SqlConnection()).ColumnOrderHints;
+
+        private static SqlBulkCopyColumnOrderHintCollection CreateCollection(params SqlBulkCopyColumnOrderHint[] orderHints)
+        {
+            Debug.Assert(orderHints != null);
+
+            SqlBulkCopyColumnOrderHintCollection collection = CreateCollection();
+
+            foreach (SqlBulkCopyColumnOrderHint orderHint in orderHints)
+            {
+                Debug.Assert(orderHints != null);
```

Comment:
It is not necessary to use this in test units if you know what is exactly happening. (optional)
But if you would rather to have it this is an extra code.